### ISSUE TITLE
Макрет битрикса

### ISSUE THREAD

**Issue by bupyc9** [worksolutions/bitrix-module-tools#23]
В маркете, у данного модуля находятся следующие файлы
.git, .gitignore, README.md, LICENSE
Предлагаю удалить данные файлы из маркета.

**Comment by sokolovsky** [comment]
согласен, но README.md оставим, т.к. это документация по использованию, а не данные разработки
